cypress/e2e/po/components/shell.po.ts

```diff
@@ -9,7 +9,7 @@ export default class Shell extends ComponentPo {
 
   openTerminal() {
     // get and click on the first row's action menu button
-    cy.get(`button[data-testid="sortable-table-0-action-button"]`).click();
+    cy.get(`button[data-testid="sortable-table-0-action-button"]`).first().click();
```

Comment:
Comment only.

At some point this needs refactoring to use ActionMenuPo and getMenuItem

Reply:
Agreed